### PR TITLE
Bg/us 32 admin invoices

### PR DESCRIPTION
**Pull request by bGray88**
Adds Admin Invoices tests, index page, controller, along with the namespace usage for Admin

### CODE REVIEW

**Review by bflanagan138 — COMMENTED** [review]
Admin invoice updates well written.

File: spec/features/admin/invoices/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @invoice_1 = create(:invoice, customer: @customer_1)
+    @invoice_2 = create(:invoice, customer: @customer_1)
+    @invoice_3 = create(:invoice, customer: @customer_2)
+    @invoice_4 = create(:invoice, customer: @customer_5)
+    @invoice_5 = create(:invoice, customer: @customer_5)
+  end
+
+  describe 'basics' do
+    it 'shows links to all invoices in the db' do
+      visit '/admin/invoices'
```
Comment (bflanagan138) [inline spec/features/admin/invoices/index_spec.rb]: do we want to standardize use of Route Helpers for this project?

Comment (bGray88) [reply spec/features/admin/invoices/index_spec.rb]: Yeah. let me change that real quick.

**Review by bflanagan138 — APPROVED** [review]
Thanks for making that route helper change!

**Pull request merged**